docs/contributing.rst

```diff
@@ -159,34 +159,68 @@ Any extra texts (print statements and so on) should be removed.
      make test-3.10-no-extensions
 
 Tests coverage
```

Comment:
@Dreamsorcerer it occurred to me that the wording of "Tests coverage" is rather weird. Can you think of an improvement here? I'm leaning towards "Code coverage" or "Covering code with tests". WDYT?